manifests/m/Microsoft/Edge/97.0.1072.76/Microsoft.Edge.installer.yaml

```
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Microsoft.Edge
+PackageVersion: 97.0.1072.76
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- msedge
+Protocols:
+- http
+- https
+FileExtensions:
+- crx
+- htm
+- html
+- pdf
+- url
+ReleaseDate: 2022-01-20T00:00:00.0000000
```

```suggestion
ReleaseDate: 2022-01-20T00:00:00.0000000
```
Something happened here.